src/alfasim_score/converter/alfacase/_tests/test_convert_well_trajectory.py

```diff
@@ -7,11 +7,7 @@
 
 from alfasim_score.converter.alfacase.convert_alfacase import ScoreAlfacaseConverter
 from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
-
-
-@pytest.fixture
-def score_input_example(shared_datadir: Path) -> ScoreInputReader:
-    return ScoreInputReader(shared_datadir / "score_input_example.json")
+from alfasim_score.converter.fixtures import score_input_example
```

Comment:
Fixtures does not need to be imported. Maybe something is missing somewhere.

Reply:
Take a look at `conftest.py` from `alfatwin_base` for example

Reply:
@lincrosenbach I moved the fixtures to the `conftest.py` is that a problem with it? Is there a better place to write the fixtures for this project. I used similar approach in floweditio, and barril do the same.

Reply:
Usually the rule is `conftest.py` for local shared fixtures and `fixtures.py` for fixtures shared to external projects.

Reply:
So moved to conftest